scripts/mbedtls_dev/bignum_common.py

```diff
@@ -150,6 +152,53 @@ def get_value_pairs(cls) -> Iterator[Tuple[str, str]]:
 # END MERGE SLOT 6
 
 # BEGIN MERGE SLOT 7
+    def arch_specific(self):
+        """ Placeholder for architecture specific patching
+        """
+        pass
```

Comment:
If I understand correctly, this is still part of the `OperationCommon` class. If that is the case then this shouldn't be in the merge slot. (If something else gets merged before, this might be appended to a different class.)